boards/shields/nrf700x_nrf54h20dk/nrf700x_nrf54h20dk.overlay

```diff
@@ -76,6 +77,23 @@
 			reg = <0x126000 DT_SIZE_K(64)>;
 		};
 	};
+	cpuapp_rw_partitions: cpuapp-rw-partitions {
+		compatible = "nordic,owned-partitions", "fixed-partitions";
+		status = "okay";
+		perm-read;
+		perm-write;
+		perm-secure;
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		dfu_partition: partition@136000 {
+			reg = < 0x136000 DT_SIZE_K(676) >;
+		};
+
+		storage_partition: partition@1df000 {
+			reg = < 0x1df000 DT_SIZE_K(24) >;
```

Comment:
Due to a series of late changes in SDFW, the DTS in upstream Zephyr was not updated.
You can use the memory up to the `0x1e9000` (start the partition from `0x1e3000`)